src/components/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+
+interface IProps {
+  inverted?: boolean;
+  text: string;
+  isColorWhite?: boolean;
+  size?: boolean;
+}
+
+export const Button: FC<IProps> = ({
+  inverted = false,
+  text,
+  isColorWhite = false,
+  size = false
+}) => (
+  <button
+    className={`mt-1 h-10 w-${size ? "10" : "48"} text-${
```

Comment:
I would create the substring in a separate function so it becomes easier to read